benchmarks/django-workload/graphite/docker-graphite-statsd/README.md

```diff
@@ -0,0 +1,235 @@
+<!-- Tocer[start]: Auto-generated, don't remove. -->
+
+## Table of Contents
+
+- [Docker Image for Graphite & Statsd](#docker-image-for-graphite--statsd)
+  - [Get Graphite & Statsd running instantly](#get-graphite--statsd-running-instantly)
+  - [Quick Start](#quick-start)
+    - [Includes the following components](#includes-the-following-components)
+    - [Mapped Ports](#mapped-ports)
+    - [Mounted Volumes](#mounted-volumes)
+    - [Base Image](#base-image)
+  - [Start Using Graphite & Statsd](#start-using-graphite--statsd)
+    - [Send Some Stats](#send-some-stats)
+    - [Visualize the Data](#visualize-the-data)
+  - [Secure the Django Admin](#secure-the-django-admin)
+  - [Change the Configuration](#change-the-configuration)
+  - [Statsd Admin Management Interface](#statsd-admin-management-interface)
+  - [Secure Grafana](#secure-grafana)
+  - [Connect Grafana to Graphite](#connect-grafana-to-graphite)
+  - [A Note on Volumes](#a-note-on-volumes)
+  - [Memcached config](#memcached-config)
+  - [Additional Reading](#additional-reading)
+  - [Contributors](#contributors)
+
+<!-- Tocer[finish]: Auto-generated, don't remove. -->
+
+[![Docker Pulls](https://img.shields.io/docker/pulls/hopsoft/graphite-statsd.svg?style=flat)](https://hub.docker.com/r/hopsoft/graphite-statsd/)
+
+> __NOTICE:__ Check out the official Graphite & Statsd Docker image at https://github.com/graphite-project/docker-graphite-statsd
+
+# Docker Image for Graphite & Statsd
+
+## Get Graphite & Statsd running instantly
+
+Graphite & Statsd can be complex to setup.
+This image will have you running & collecting stats in just a few minutes.
+
+## Quick Start
+
+```sh
+docker run -d\
+ --name graphite\
+ --restart=always\
+ -p 80:80\
+ -p 81:81\
+ -p 2003-2004:2003-2004\
+ -p 2023-2024:2023-2024\
+ -p 8125:8125/udp\
+ -p 8126:8126\
+ hopsoft/graphite-statsd
```

Comment:
I think we should change this line to the cloudsuite repository (e.g., cloudsuite/django:graphite-statsd or any other proper one).

Reply:
This image is not explicitly used while running the django image. 
I have retained the original readme from hopsoft just as a reference.

This readme need not be read/used while deploying django workload